README.md

```diff
@@ -25,13 +25,20 @@ The Operator SDK is a framework that uses the [controller-runtime][controller_ru
 
 The SDK provides workflows to develop operators in Go or Ansible.
 
-The following workflow is for a new Go operator:
+The following workflow is for a new **Go** operator:
 1. Create a new operator project using the SDK Command Line Interface(CLI)
 2. Define new resource APIs by adding Custom Resource Definitions(CRD)
 3. Define Controllers to watch and reconcile resources
 4. Write the reconciling logic for your Controller using the SDK and controller-runtime APIs
 5. Use the SDK CLI to build and generate the operator deployment manifests
 
+The following workflow is for a new **Ansible** operator:
```

Comment:
Don't steps 1, 2, 4 and 5 all largely happen as the result of the `operator-sdk` new command? I know it only gives you a single CRD to work with, but that will get most people a long way when they start. I'd describe the workflow more as:

1. create new operator with the sdk
2. write ansible to do stuff
3. use sdk to build and run the operator
4. optionally add additional CRDs for specific purposes

Reply:
I agree I am going to change better align with @mhrivnak's feedback